src/Features/Core/Portable/EditAndContinue/EditSession.cs

```diff
@@ -869,13 +869,13 @@ public async Task<SolutionUpdate> EmitSolutionUpdateAsync(Solution solution, Sol
                     // since we already checked that no changed document is out-of-sync above.
                     var oldActiveExceptionRegions = await GetBaseActiveExceptionRegionsAsync(solution, cancellationToken).ConfigureAwait(false);
 
-                    var lineEdits = await projectChanges.LineChanges.SelectAsArrayAsync(async lineChange =>
+                    var lineEdits = await projectChanges.LineChanges.SelectAsArrayAsync(async (lineChange, cancellationToken) =>
```

Comment:
The signature of `SelectAsArrayAsync` changed, so this was updated to match.

Reply:
I guess I wasn't clear -- what's the benefit of changing the signature?

Reply:
It allows the asynchronous operation to be cancelled.

Reply:
So that was fine before given the cancellation token isn't actually used in the implementation, it'll just pass through an cancellation observed. Did you want to add a use of the token in the loop then?

Reply:
It already existed. Same token is used in the new form, only now it's not captured.